cli-plugins/apps/reference/clustersupplychain.hbs.md

```diff
@@ -2,16 +2,17 @@
 
 This command provides details about the cluster supply chain.
```

Comment:
This command provides details about the cluster supply chain(s) installed on the target cluster.

Reply:
changed